### DISCUSSION TITLE
[Feature] improve the selection of backup folders (upload from mobile)

### DISCUSSION THREAD

**Discussion by aisbergde** [immich-app/immich#6185] [Feature Request]
### The feature

- I think nearly nobody wants to upload all assets from all folders => do not list the "Recent" (Android)
- pre select the camera as default, most users want to save it and only it
- maybe: list selected folders on top of not selected folders, they could be hard to find, if there are a lot of potential folders
- maybe: show a hierarchical presentation of folders, because some apps create a whole hierarchy of folders containing possible assets, we don't want to backup.
- if folders are showing in a flat list, like now, which is also fine, then there should be an easy way (easy for non nerds, but for normal users, not knowing special Unix command syntax), to exclude folders and their subfolders. A kind of file browser should be used to select folders to be excluded together with their subfolders.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5148
